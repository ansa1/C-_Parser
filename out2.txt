[Keyword -> "using"] [Identifier -> "System"] [Delimiter -> ";"] 
[Keyword -> "class"] [Identifier -> "sample"] 
[Delimiter -> "{"] 
[Keyword -> "public"] [Keyword -> "static"] [Keyword -> "void"] [Identifier -> "Main"] [Operator -> "()"] 
[Delimiter -> "{"] 
[Keyword -> "int"] [Identifier -> "x"] [Operator -> "="] [Numerical constant -> "1024"] [Operator -> "*"] [Numerical constant -> "1024"] [Operator -> "*"] [Numerical constant -> "1024"] [Delimiter -> ";"] 
[Keyword -> "uint"] [Identifier -> "p"] [Operator -> "="] [Numerical constant -> "1024"] [Operator -> "*"] [Numerical constant -> "1024"] [Operator -> "*"] [Numerical constant -> "1024"] [Delimiter -> ";"] 
[Keyword -> "int"] [Identifier -> "y"] [Operator -> "="] [Numerical constant -> "-42"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "WriteLine"] [Operator -> "("] [Literal constant -> "LEFT SHIFT OPERATIONS :"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "WriteLine"] [Operator -> "("] [Literal constant -> "{0},{1},{2}"] [Operator -> ","] [Identifier -> "x"] [Operator -> ","] [Identifier -> "x"] [Operator -> "*"] [Numerical constant -> "2"] [Operator -> ","] [Identifier -> "x"] [Operator -> "<<"] [Numerical constant -> "1"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "WriteLine"] [Operator -> "("] [Literal constant -> "{0},{1},{2}"] [Operator -> ","] [Identifier -> "p"] [Operator -> ","] [Identifier -> "p"] [Operator -> "*"] [Numerical constant -> "2"] [Operator -> ","] [Identifier -> "p"] [Operator -> "<<"] [Numerical constant -> "1"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "WriteLine"] [Operator -> "("] [Literal constant -> "{0},{1},{2}"] [Operator -> ","] [Identifier -> "x"] [Operator -> ","] [Identifier -> "x"] [Operator -> "*"] [Numerical constant -> "4"] [Operator -> ","] [Identifier -> "x"] [Operator -> "<<"] [Numerical constant -> "2"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "WriteLine"] [Operator -> "("] [Literal constant -> "{0},{1},{2}"] [Operator -> ","] [Identifier -> "p"] [Operator -> ","] [Identifier -> "p"] [Operator -> "*"] [Numerical constant -> "4"] [Operator -> ","] [Identifier -> "p"] [Operator -> "<<"] [Numerical constant -> "2"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "WriteLine"] [Operator -> "("] [Literal constant -> "{0},{1},{2}"] [Operator -> ","] [Identifier -> "y"] [Operator -> ","] [Identifier -> "y"] [Operator -> "*"] [Numerical constant -> "1024"] [Operator -> "*"] [Numerical constant -> "1024"] [Operator -> "*"] [Numerical constant -> "64"] [Operator -> ","] [Identifier -> "x"] [Operator -> "<<"] [Numerical constant -> "26"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "ReadLine"] [Operator -> "()"] [Delimiter -> ";"] 
[Delimiter -> "}"] 
[Delimiter -> "}"] 