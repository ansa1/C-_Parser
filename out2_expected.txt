[Keyword -> "using"] [Identifier -> "System"] [Delimiter -> ";"] 
[Keyword -> "class"] [Identifier -> "program"] 
[Delimiter -> "{"] 
[Keyword -> "public"] [Keyword -> "static"] [Keyword -> "void"] [Identifier -> "Main"] [Operator -> "()"] 
[Delimiter -> "{"] 
[Keyword -> "int"] [Identifier -> "i"] [Operator -> ","] [Identifier -> "k"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "WriteLine"] [Operator -> "("] [Literal constant -> "Enter two integers \n"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "i"] [Operator -> "="] [Keyword -> "int"] [Operator -> "."] [Identifier -> "Parse"] [Operator -> "("] [Identifier -> "Console"] [Operator -> "."] [Identifier -> "ReadLine"] [Operator -> "()"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "k"] [Operator -> "="] [Keyword -> "int"] [Operator -> "."] [Identifier -> "Parse"] [Operator -> "("] [Identifier -> "Console"] [Operator -> "."] [Identifier -> "ReadLine"] [Operator -> "()"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "WriteLine"] [Operator -> "("] [Literal constant -> "\n Before swapping i= {0} and k = {1}"] [Operator -> ","] [Identifier -> "i"] [Operator -> ","] [Identifier -> "k"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "i"] [Operator -> "="] [Identifier -> "i"] [Operator -> "^"] [Identifier -> "k"] [Delimiter -> ";"] 
[Identifier -> "k"] [Operator -> "="] [Identifier -> "i"] [Operator -> "^"] [Identifier -> "k"] [Delimiter -> ";"] 
[Identifier -> "i"] [Operator -> "="] [Identifier -> "i"] [Operator -> "^"] [Identifier -> "k"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "WriteLine"] [Operator -> "("] [Literal constant -> "\n After swapping i= {0} and k = {1}"] [Operator -> ","] [Identifier -> "i"] [Operator -> ","] [Identifier -> "k"] [Operator -> ")"] [Delimiter -> ";"] 
[Identifier -> "Console"] [Operator -> "."] [Identifier -> "ReadLine"] [Operator -> "()"] [Delimiter -> ";"] 
[Delimiter -> "}"] 
[Delimiter -> "}"] 